db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171015203318) do
```

Comment:
Teremos problemas se integrar o schema no master. Eu fui integrar com o master e tive conflito entre as versões. Andei pesquisando formas de integrar o `schema.rb` sem que dê problemas nas versões do banco diferentes entre branches.

Acho que você terá que obter esta versão do master e integrar com ela desatualizada para que a `migration` que você criou possa rodar em produção. Se atualizar a versão aqui acho que ele vai dar problema em produção quando atualizar a versão do schema, pois não vai considerar que tem m`migrations` pendentes.

Se souber de algo ou se já teve este problema fique a vontade para sugerir uma melhor solução

Reply:
Olá, @alexanmtz . Acredito que a melhor saída seria em **development** manter o "controle de conflitos" na version da `schema.rb` (isso já acontece) baseado na versão de maior número em todos os momentos. [Ver Migrations in source control](http://edgeguides.rubyonrails.org/active_record_migrations.html#schema-dumps-and-source-control)
Assim quando em **production** for rodado o esquema de carregar as migrations vai ser verificado que há uma versão maior em `schema.rb` e portanto precisa carregar `migrations` pendentes, e isso será feito em ordem devido a versão que cada `migration` tem em seu nome de arquivo. 
> It will run these migrations in order based on the date of the migration
> - (Ver Running Migrations)[http://edgeguides.rubyonrails.org/active_record_migrations.html#running-migrations)

Então, em resumo, acredito que o ideal é **mergear** a branch com a master > corrigir a versão da `schema.rb` para a que tiver maior valor e em production rodar o `db:migrate` normalmente.